gci/models.py

```diff
@@ -1 +1,87 @@
-# Create your models here.
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from django.db import models
+
+
+class Students(models.Model):
+
+    identifier = models.IntegerField(default=0, primary_key=True)
+    name = models.CharField(max_length=100)
+    program_year = models.ForeignKey('ProgramYear', on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.student_name
+
+
+class Organizations(models.Model):
+
+    identifier = models.IntegerField(default=0, primary_key=True)
+    name = models.CharField(max_length=100)
+    logo = models.ImageField(upload_to='', blank=True)
+    url = models.URLField(blank=True)
+    summary = models.TextField(max_length=300, default='')
+    students = models.ManyToManyField(Students)
+
+    def __str__(self):
+        return self.org_name
+
+
+class Tasks(models.Model):
+
+    name = models.CharField(max_length=200)
+    description = models.TextField(max_length=500)
+    status = models.IntegerField(default=1)
+    max_instances = models.IntegerField(default=1)
+    mentors = models.ManyToManyField('Mentors')
+    tags = models.ManyToManyField('Tags')
+    is_beginner = models.BooleanField(default=False)
+    categories = models.ManyToManyField('Categories')
+    time_to_complete_in_days = models.IntegerField(default=3)
+    external_url = models.URLField()
+    metadata = models.CharField(max_length=80)
+    last_modified = models.DateField()
+    org = models.ForeignKey(Organizations, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.name
+
+
+class TaskInstance(models.Model):
+
+    task = models.ForeignKey(Tasks, on_delete=models.CASCADE)
+    student = models.ForeignKey(Students, on_delete=models.CASCADE)
+    organization = models.ForeignKey(Organizations, on_delete=models.CASCADE)
+    status = models.IntegerField()
```

Comment:
Maybe it's better having a choice based field here, because the output are limited here.